c_binding/include/libstoragemgmt/libstoragemgmt.h

```diff
@@ -1029,6 +1030,102 @@ int LSM_DLL_EXPORT lsm_volume_ident_led_set(lsm_connect *c,
 int LSM_DLL_EXPORT lsm_volume_ident_led_clear(lsm_connect *c,
                                               lsm_volume *volume,
                                               lsm_flag flags);
+/**
+ * Get a list of batteries for this storage system.
+ * The supper capacitor will also be included.
```

Comment:
*super

Maybe change the whole line to something like: "When present, super capacitors will also be included."